### PR TITLE
Try and Ececption

### PR DESCRIPTION
**Pull request by Iihtisham12**
I Add Some Try and Ececption for Some reason

### CODE REVIEW

File: ihtishamul.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# This is Ihtisham Baranch
+
+print("You are in Ihtisham")
+
+# Try and Expation
+name = input("Enter Ur Name!\n")
+try:
```
Comment (ariguli) [inline ihtishamul.py:7]: change this function to sum function

Comment (Iihtisham12) [reply ihtishamul.py:7]: What Can I Change

Comment (Iihtisham12) [reply ihtishamul.py:7]: Thank U For Ur Response

**Review by ariguli — APPROVED** [review]

**Review by ariguli — APPROVED** [review]

File: ihtishamul.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    if int(name) // int(name) == 1:
```
Comment (mhassan-mustafa) [inline ihtishamul.py:8]: This statement `int(name) // int(name) == 1` seems so complicated, you can add simple condition as follows:
`type(name) == str`

Comment (Iihtisham12) [reply ihtishamul.py:8]: Thank U But It's Write a Simple Code If u Want to i Change This

Comment (mhassan-mustafa) [reply ihtishamul.py:8]: Please change it...

Comment (mhassan-mustafa) [reply ihtishamul.py:8]: `name = input("Please write your name: ")

if type(name) == str:
    if name.isdigit():
        print("Number not allowed")
    else:
        print(name.capitalize())
        print(type(name))
else:
    print("Number not allowed")`

**Review by mhassan-mustafa — APPROVED** [review]

**Pull request merged**